workers/shareScroll.worker.ts

```diff
@@ -432,33 +458,36 @@ async function getDragonStrip(dragonBuffers: Buffer[]) {
 }
 
 async function createFrames(
-  bannerBuffer: Buffer,
+  bannerFrames: Buffer[],
+  delays: number[],
   stripBuffer: Buffer,
   stripWidth: number,
   stripHeight: number
 ) {
   const framePromises: Array<Promise<sharp.Sharp> | sharp.Sharp> = [];
-
   if (stripWidth < baseCarouselWidth) {
-    const composedFrameBuffer = createEmptyFrame(
-      baseBannerWidth,
-      baseBannerHeight
-    ).composite([
-      {
-        input: bannerBuffer,
-        top: 0,
-        left: 0,
-      },
-      {
-        input: stripBuffer,
-        top: 5,
-        left:
-          Math.floor(baseCarouselWidth / 2) - Math.floor(stripWidth / 2) + 5,
-      },
-    ]);
-
-    framePromises.push(composedFrameBuffer);
+    await Promise.all(
+      bannerFrames.map((buffer) => {
+        const composedFrameBuffer = createEmptyFrame(
+          baseBannerWidth,
+          baseBannerHeight
+        ).composite([
+          { input: buffer, top: 0, left: 0 },
+          {
+            input: stripBuffer,
+            top: 5,
+            left:
+              Math.floor(baseCarouselWidth / 2) -
+              Math.floor(stripWidth / 2) +
+              5,
+          },
+        ]);
+        framePromises.push(composedFrameBuffer);
+      })
+    );
   } else {
+    let msElapsed = 0;
+    let currentPos = 0;
```

Comment:
```suggestion
    let currentPosition = 0;
```

this suggestion just makes the name consistent as we already have `scrollPosition`